test/test.googleauth.ts

```diff
@@ -402,6 +402,20 @@ describe('googleauth', () => {
       assert.strictEqual(json.private_key, (result as JWT).key);
     });
 
+    it.only('fromJSON should set default access variable with private key', () => {
```

Comment:
blocking so we don't merge with the `.only`

Reply:
yep, sorry still haven't pushed the new changes!